docs/source/doctest_fixtures.py

```diff
@@ -85,6 +85,7 @@
     sys.path.insert(0, _repo_root)
 
 from tests.common import SimpleModel
+from tests.common.datasets import RandomTextClassificationDataset
```

Comment:
unused line?

Reply:
Not unused, need to import here so it is accessible in doctests